Kickstarter-iOS/Features/Comments/Controllers/CommentRepliesViewController.swift

```diff
@@ -231,6 +252,15 @@ extension CommentRepliesViewController: RootCommentCellDelegate {
   }
 }
 
+// MARK: - MessageBannerViewControllerDelegate
+
+extension CommentRepliesViewController: MessageBannerViewControllerDelegate {
+  func messageBannerViewDidHide(type _: MessageBannerType) {
+    self.commentComposer.isHidden = false
+    self.view.isUserInteractionEnabled = true
```

Comment:
I took a peek in `MessageBannerViewController`, and it does already reference `self.view.superview`, so it might be safe to move `isUserInteractionEnabled` in there. But that's not a blocker, as previously discussed.